modules/audio/src/main/native/android/dalvik/DalvikAudioService.java

```diff
@@ -42,8 +43,8 @@ public class DalvikAudioService {
 
     private SoundPool pool = null;
 
-    public DalvikAudioService() {
-
+    public DalvikAudioService(Activity activity) {
+        activity.setVolumeControlStream(AudioManager.STREAM_MUSIC);
```

Comment:
Testing on Android, I see that pressing the volume buttons has some effect _before_ the service is called, and then while audio is playing, I don't see any effect. Is this expected?

For the videoService, we also used the AudioManager:
https://github.com/gluonhq/attach/blob/master/modules/video/src/main/native/android/dalvik/DalvikVideoService.java#L118 

and a keyEvent dispatcher:
https://github.com/gluonhq/attach/blob/master/modules/video/src/main/native/android/dalvik/DalvikVideoService.java#L725

Do you think we need it here too?